closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -314,6 +319,22 @@ int encrypt(byte[] in, int inOfs, int len, byte[] out, int outOfs) {
         return 0;
     }
 
+    /*
+     * This method is for CipherCore to insert the remainder of its buffer
+     * into the ibuffer before a encryptFinal(ByteBuffer, ByteBuffer) operation
+     */
+    int encrypt(byte[] in, int inOfs, int len) {
+        if (len > 0) {
+            // store internally until encryptFinal
+            ArrayUtil.nullAndBoundsCheck(in, inOfs, len);
+            if (ibuffer == null) {
+                ibuffer = new ByteArrayOutputStream();
```

Comment:
The comment on `ibuffer` says 'not used for encryption'; perhaps you meant `ibuffer_enc`?

Reply:
Yes, I meant `ibuffer_enc`. Fixed it.